sublime_tensorflow_autocomplete.py

```diff
@@ -2090,13 +2090,38 @@ class TensorflowDocCommand(sublime_plugin.TextCommand):
 
     def run(self, edit):
 
+        is_auto_selection = False
         selection = ""
-        for region in self.view.sel():
-            selection += self.view.substr(region)
+        regions = self.view.sel()
 
-        selec_link = selection.replace('.', '/')
+        if len(regions) == 1 and len(regions[0]) == 0:
+            selection = self.view.substr(
+                self.extend_point_by_allowed_chars(
+                    point=regions[0].a,
+                    allowed_chars='0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ._',
+                )
+            )
+            is_auto_selection = True
+        else:
+            for region in self.view.sel():
+                selection += self.view.substr(region)
 
         if selection + "()" in tensorflow_functions:
+            selec_link = selection.replace('.', '/')
             webbrowser.open("https://www.tensorflow.org/api_docs/python/%s" % selec_link)
-        else:
-            sublime.error_message("Not a Tensorflow class or function.\nHere is an example of what can be selected : 'tf.nn.conv2d'")
+        elif not is_auto_selection:
```

Comment:
This means the error message will only be displayed when there is a selection. It's preferable to be displayed even if there is no selection?